tests/cypress/integration/reportRepair/smellGas.spec.js

```diff
@@ -0,0 +1,11 @@
+describe('smellGas', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/report-repair/smell-gas');
+  })
+  it('Displays non-mvp title', () => {
+    cy.contains('What to do if you smell gas');
+    cy.get('li').contains('put out all naked flames');
+    cy.get('li').contains('open doors and windows');
```

Comment:
There are tests for some of the content (these 2 bullet points), but not others (i.e. other bullet points, including those from the 'Do not' section).

Is there a reason for this level of tests?
Are there plans to add more tests?